fxa-auth-db-server/test/backend/db_tests.js

```diff
@@ -1208,6 +1211,36 @@ module.exports = function(config, DB) {
                 t.equal(device.callbackPublicKey, '', 'callbackPublicKey updated')
                 t.equal(device.callbackAuthKey, '', 'callbackAuthKey updated')
 
+                // Make the device a zombie, by giving it a non-existent session token
+                return db.updateDevice(ACCOUNT.uid, deviceId, {
+                  sessionTokenId: zombieSessionTokenId
+                })
+                .catch(function () {
+                  t.fail('updating an existing device should not have failed')
+                })
+              })
+              .then(function () {
+                // Fetch all of the devices for the account
+                return db.accountDevices(ACCOUNT.uid)
+              })
+              .then(function (devices) {
+                t.equal(devices.length, 0, 'devices is empty')
```

Comment:
Should we update the documentation for [accountDevices](https://github.com/mozilla/fxa-auth-db-mysql/blob/master/docs/API.md#accountdevicesuid) to mention that devices with zombie session wouldn't be returned.